bin/deployment/deploy

```diff
@@ -384,13 +384,6 @@ fi
 # deploy the webapp
 deploy
 
-# Starting with jss 5, the jar is called 'jss.jar' instead of 'jss4.jar'
-if [ -f "/usr/lib64/jss/jss.jar" ]; then
-    $SUDO unzip -q $DEPLOY -d $DEPLOY_DIR
-    $SUDO sed -i 's/jss4.jar/jss.jar/g' $DEPLOY_DIR/META-INF/context.xml
```

Comment:
btw, you also need to update the spec file (candlepin.spec.tmpl) because it's doing the same kind of context.xml manipulation when building the RPM. Looking at the brew error from the jenkins release job from yesterday, it failed because it couldn't find context.xml